go/pkg/rexec/rexec.go

```diff
@@ -19,23 +20,15 @@ import (
 	log "github.com/golang/glog"
 )
 
-// FileDigestCache is a cache for file contents->digests.
-type FileDigestCache interface {
-	Get(string) (digest.Digest, error)
-}
-
-// NoopFileDigestCache is a non-caching cache (always returns a cache miss).
-type NoopFileDigestCache struct{}
-
-// Get computes the digest from the file contents.
-func (c *NoopFileDigestCache) Get(path string) (digest.Digest, error) {
-	return digest.NewFromFile(path)
+// FileMetadataCache is a cache for file contents->Metadata.
+type FileMetadataCache interface {
+	Get(string) (*filemetadata.Metadata, error)
```

Comment:
Does golint not complain about Get not being documented?

Reply:
Apparently not! :-/